packages/reports/tests/unit/routes/reports/new-test.js

```diff
@@ -147,11 +147,7 @@ module('Unit | Route | reports/new', function(hooks) {
         tableTimeGrains = A(get(table, 'timeGrains')),
         timeGrainName = this.owner.lookup('route:reports/new')._getDefaultTimeGrainName(table);
 
-      assert.deepEqual(
-        timeGrainName,
-        get(tableTimeGrains, 'firstObject.name'),
-        'Return the first time grain in the table'
-      );
+      assert.deepEqual(timeGrainName, 'day', 'Return the default time grain from config');
```

Comment:
Can we get the expected timegrain from the config itself rather than using a hardcoded string?